hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AzureBlobFileSystemStore.java

```diff
@@ -817,8 +817,8 @@ public AbfsInputStream openFileForRead(Path path,
       FileStatus fileStatus = parameters.map(OpenFileParameters::getStatus)
           .orElse(null);
       String relativePath = getRelativePath(path);
-      String resourceType, eTag;
-      long contentLength;
+      String resourceType = null, eTag = null;
```

Comment:
explicit null initialization not needed

Reply:
to prevent compilation error. As these two variables are used to create inputStream. Now, these variables may or may not be initialized, hence we need to set as null in the starting. In case if it would be that these variables were always initialized, we would not need to set as null in the starting.